modules/ossm-cert-manager-integration.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration_{context}"]
+== Integrating cert-manager with Service Mesh
+
+cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. 
```

Comment:
Please also add a note about istio-csr. Look at the docs: https://cert-manager.io/docs/projects/istio-csr/.

Reply:
In short: istio-csr acts as an external CA for Istio and when it's enabled, apps within a mesh, send requests for signing certificates to istio-csr, not to istiod.